test/testdata/fixtures/bitbucket/webhook-create.json

```diff
@@ -0,0 +1,15 @@
+{
+    "id": 10,
+    "name": "Webhook Name",
+    "createdDate": 1513106011000,
+    "updatedDate": 1513106011000,
+    "events": [
+        "repo:refs_changed",
+        "repo:modified"
+    ],
```

Comment:
Shouldn't we also include some more events like pull requested opened, edited and, declined?

Reply:
These files are exact copies of the Bitbucket REST API examples. In any case this is just to test the API interaction so it does not really matter what values we use here ...